cbor_cddl_analyzer.py

```
@@ -327,15 +371,17 @@ def parse(self):
             
             # Handle group definitions (name = ( ... ))
             # Note: Groups can span multiple lines, ending with )
-            # Must NOT confuse with IANA parameters &( ... )
+            # Must NOT confuse with IANA parameters &( ... ) or annotations like .size (M..N)
             if '=' in line and '(' in line and not line_normalized.startswith('&') and not '{' in line and not '[' in line and '/=' not in line and '#6.' not in line:
```

Ruff (0.15.1)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)